app/src/main/java/com/github/se/polyfit/di/UserModule.kt

```diff
@@ -63,6 +64,12 @@ object UserModule {
     return MealRepository(context, mealFirebaseRepository, mealDao)
   }
 
+  @Provides
+  @Singleton
+  fun providePostFirebaseRepository(): PostFirebaseRepository {
```

Comment:
It seems like this function is not used anywhere right now, is it for future uses?